README.md

```diff
@@ -23,7 +23,6 @@ Legend:
 
 ## General
 
-  - Console*[no-console]: console.log should be removed whenever the code will be pushed
```

Comment:
I would keep a note on this on the README or somewhere, to discuss later a good logging solution. I'm keen on adding this rule as soon as we have some time.